configuration/env.js

```diff
@@ -10,5 +10,6 @@ module.exports = {
     SMTP_PORT: process.env.SMTP_PORT,
     SMTP_SECURE: process.env.SMTP_SECURE === 'true',
     SMTP_USER: process.env.SMTP_USER,
-    SMTP_PASS: process.env.SMTP_PASS
+    SMTP_PASS: process.env.SMTP_PASS,
+    RESET_PASSWORD_URL: `${values.FRONTEND_URL}/reset-password/`
```

Comment:
You don't need to keep this here. Keep just frontend url. And i don't see here import of FRONTEND_URL. Did you test it. Is this work